sdk/storage/storage-queue/src/QueueServiceClient.ts

```diff
@@ -215,6 +220,34 @@ export class QueueServiceClient extends StorageClient {
    *                                                  AnonymousCredential is used.
    * @param {StoragePipelineOptions} [options] Options to configure the HTTP pipeline.
    * @memberof QueueServiceClient
+   *
+   * @example
+   * ```js
+   * const account = "<account>";
+   *
+   * const credential = new DefaultAzureCredential();
```

Comment:
Can we add an inline comment about where to get `DefaultAzureCredential`? It's not from storage SDK.